client/me/purchases/remove-purchase/index.jsx

```diff
@@ -317,24 +319,23 @@ const RemovePurchase = React.createClass( {
 						this.state.survey.questionTwoRadio === null ||
 						( this.state.survey.questionOneRadio === 'anotherReasonOne' && this.state.survey.questionOneText === '' ) ||
 						( this.state.survey.questionTwoRadio === 'anotherReasonTwo' && this.state.survey.questionTwoText === '' ),
-					label: this.translate( 'Next' ),
+					label: translate( 'Next Step' ),
 					onClick: this.changeSurveyStep
 				},
 				prev: {
 					action: 'prev',
 					disabled: this.state.isRemoving,
-					label: this.translate( 'Previous' ),
-					onClick: this.changeSurveyStep.bind( null, 'previous' )
+					label: translate( 'Previous Step' ),
+					onClick: this.changeSurveyStep
```

Comment:
This breaks the previous button when the business onboarding offer is completed. When there are 3 survey steps total, `changeSurveyStep` needs a direction do know where to go from step 2. Removing `.bind( null, 'previous' )` here means you always get directed to step 3 when you're on step 2 even if you hit previous.